src/pages/Home.tsx

```diff
@@ -1,12 +1,25 @@
 import React from 'react'
+import { Route, Switch } from 'react-router-dom'
 import { ButtonNav } from '../components/ButtonNav'
 import Header from '../components/Header'
+import Matching from './Matching'
+import Profile from './Profile'
 
 const Home: React.FC = () => {
 	return (
 		<>
 			<Header />
-			<ButtonNav />
+			<Switch>
```

Comment:
`Switch` and `Route` shouldn't be outside of App component, if I'm not mistaken. 

I thought this is just general page user will see after coming on our webpage for the first time, with the login/registration button?

Reply:
Ok, I didn't know that the switch was only used in the App component.

I was inspired here: https://reactrouter.com/web/example/nesting 
What is the alternative for changing the content inside?

Reply:
For changing content inside, yes, it confused me because I didn't expected it. 

I don't know how to link the buttons to where they're supposed to be leading in React but I don't think it's Router issue we should be solving in Home page. :thinking: I might be mistaken here though. @macajind